pkg/auth/config/identity.go

```diff
@@ -235,16 +245,36 @@ type OAuthSSOProviderConfig struct {
 	// KeyID and TeamID are specific to apple
 	KeyID  string `json:"key_id,omitempty"`
 	TeamID string `json:"team_id,omitempty"`
+
+	Scope string `json:"-"`
```

Comment:
I suppose we should move the scopes to the specific providers, since we're not going to allow developer to configure them?